.changeset/brave-impalas-switch.md

```diff
@@ -0,0 +1,24 @@
+---
+'example-backend': patch
+'@backstage/backend-common': patch
+'@backstage/config-loader': patch
```

Comment:
It's typically nicer to create multiple changesets when the changes differ, in order to avoid noise in the changelog of each package, which is where this text ultimately ends up